advanced/myserverextension/package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "jupyterlab_myserverextension",
+  "version": "0.0.0",
+  "description": "A minimal JupyterLab extension with backend and frontend parts.",
+  "keywords": [
+    "jupyter",
+    "jupyterlab",
+    "jupyterlab-extension"
+  ],
+  "homepage": "https://github.com/my_name/myextension",
+  "bugs": {
+    "url": "https://github.com/my_name/myextension/issues"
+  },
+  "license": "BSD-3-Clause",
+  "author": "my_name",
+  "files": [
+    "lib/**/*.{d.ts,eot,gif,html,jpg,js,js.map,json,png,svg,woff2,ttf}",
+    "style/**/*.{css,eot,gif,html,jpg,json,png,svg,woff2,ttf}"
+  ],
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "style": "style/index.css",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/my_name/myextension.git"
+  },
+  "scripts": {
+    "build": "jlpm run build:lib",
+    "build:labextension": "cd myserverextension && rimraf labextension && mkdirp labextension && cd labextension && npm pack ../..",
+    "build:lib": "tsc",
+    "build:all": "jlpm run build:labextension",
+    "clean": "jlpm run clean:lib",
+    "clean:lib": "rimraf lib tsconfig.tsbuildinfo",
+    "clean:labextension": "rimraf myserverextension/labextension",
+    "clean:all": "jlpm run clean:lib && jlpm run clean:labextension",
+    "prepare": "jlpm run clean && jlpm run build",
```

Comment:
Should we also add the `link` script like for the extensions, so everything can be built and installed at once with lerna?

Reply:
Yes and I realized I forgot to do so in the other examples.